docs/DeveloperGuide.md

```diff
@@ -363,22 +363,22 @@ Step 3. CONNECTIONS displays a list of available commands in `ResultDisplay`.
 
 Step 4. The user decides to view the usage of `add` to learn to add a contact, and executes `help add`.
 
-Step 5. CONNECTIONS will display a detailed help message on the usage of `add` command in `ResultDisplay`.
+Step 5. CONNECTIONS will display a detailed help message on the usage of the `add` command in `ResultDisplay`.
 
 
 ### Birthday Reminder feature
 
 #### Implementation
 
-Shows a list of contacts with upcoming birthdays. This list of birthday reminders is displayed to the user though the `UI`, specifically in `BirthdayReminderListPanel`. Each birthday is displayed as a `BirthdayReminderCard`.
+Shows a list of contacts with upcoming birthdays. This list of birthday reminders is displayed to the user through the `UI`, specifically in `BirthdayReminderListPanel`. Each birthday is displayed as a `BirthdayReminderCard`.
 The list of birthdays is generated in the `ModelManager`, which implements the following functions:
 * `ModelManager#getBirthdayReminderList` returns an `ObservableList<Person> birthdayReminders` that is ordered according to upcoming birthdays.
 
 Given below is an example usage scenario and how the birthday reminders mechanism behaves at each step.
 
-Step 1. On app startup sort contacts with birthday by birth month and day only. Contacts with birthdays that fall on the present day are coloured green while birthdays that are within one week are coloured blue. Additionally, a customised message with each contact's phone number and age is displayed in the birthday reminder list.
+Step 1. On app startup sort contacts with a birthday by birth month and day only. Contacts with birthdays that fall on the present day are coloured green while birthdays that are within one week are coloured blue. Additionally, a customised message with each contact's phone number and age will be displayed in the birthday reminder list.
```

Comment:
Will `On app startup, contacts with birthday are sorted by birth month and day.` be better?